packages/eslint-plugin-next/lib/rules/no-document-import-in-page.js

```diff
@@ -16,7 +16,11 @@ module.exports = {
         const paths = context.getFilename().split('pages')
         const page = paths[paths.length - 1]
 
-        if (!page || page.startsWith('/_document')) {
+        if (
+          !page ||
+          page.startsWith('/_document') ||
+          page.startsWith('\\_document')
```

Comment:
can we use `path.sep` and `path.posix.sep` instead of literal slashes?

```
page.startsWith(`${path.sep}_document`) || page.startsWith(`${path.posix.sep}_document`)
```